README.md

```diff
@@ -37,7 +39,7 @@ Automatically adds roles to the channels created by the tracked users.
  Here are the steps to host your copy of this bot:
 
 1. Clone or Download the repo to your machine of choice.
-2. Run `pip install -r requirements.txt`
+2. Run (Assuming pip and python are already on the machine) `pip install -r requirements.txt`
```

Comment:
If you're making it more specific, maybe go and do a venv setup info?

Reply:
> If you're making it more specific [..] 
Didn't intend on making it *too* specific, just wanted to add the detail to clear anything that could possibly come up in the future.  

> [..] maybe go and do a venv setup info?
What do you mean on that?

Reply:
> What do you mean by that?

Maybe a rough guide to python instance separation to avoid dependency conflicts.
Considering venv is included with python by default.
python -m venv 'virt environment name' etc.

Reply:
> > What do you mean by that?
> 
> Maybe a rough guide to python instance separation to avoid dependency conflicts. Considering venv is included with python by default. python -m venv 'virt environment name' etc.

Is venv required for building? I was just making sure to mention the requirement of python+pip, If venv is reuqired with python by default (and needed). This should be assumed to be included when the end user would to install it.

Reply:
Isn't required but I would say definitely **recommended** to avoid dependency conflicts.

Reply:
Before. And after creating venv do /'venv name here'/scripts/activate
To actually use it